core/controllers/story_editor.py

```diff
@@ -34,6 +34,17 @@
 class StoryEditorPage(base.BaseHandler):
     """The editor page for a single story."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'story_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
```

Comment:
Should we impose more rules on `story_id`?

Reply:
ditto elsewhere

Reply:
I think we can use this as a reference.

![Screenshot from 2021-09-24 22-44-47](https://user-images.githubusercontent.com/42772952/134715052-76ecdd58-047b-4953-aa1f-605b80e31e94.png)

Reply:
Agreed